components/webext-storage/src/sync/bridge.rs

```diff
@@ -5,18 +5,30 @@
 use anyhow::Result;
 use rusqlite::Transaction;
 use std::sync::{Arc, Weak};
-use sync15::bso::IncomingBso;
-use sync15::engine::ApplyResults;
+use sync15::bso::{IncomingBso, OutgoingBso};
+use sync15::engine::{ApplyResults, BridgedEngine as Sync15BridgedEngine};
 use sync_guid::Guid as SyncGuid;
 
 use crate::db::{delete_meta, get_meta, put_meta, ThreadSafeStorageDb};
 use crate::schema;
 use crate::sync::incoming::{apply_actions, get_incoming, plan_incoming, stage_incoming};
 use crate::sync::outgoing::{get_outgoing, record_uploaded, stage_outgoing};
+use crate::WebExtStorageStore;
 
 const LAST_SYNC_META_KEY: &str = "last_sync_time";
 const SYNC_ID_META_KEY: &str = "sync_id";
 
+impl WebExtStorageStore {
+    // Returns a bridged sync engine for Desktop for this store.
```

Comment:
total nit, but "for Desktop" will probably not be true once we expose this to mobile - I'd just remove those 2 words.